### ISSUE TITLE
Channel Points not visible when going into a new stream when claim a bonus is visible

### ISSUE THREAD

**Issue by vishwaschoudhary** [twitchdev/issues#967]
**Brief description**
While going into a stream users are not able to see the channel points if the "Claim A Bonus" is available Bits icon with 0 next to it will be visible but at the same time the channel points gets invisible. The asset that shows the channel points shows as visible in the console.

**How to reproduce**
Join a stream and wait for some time for bonus points to get active.

**Expected behaviour**
Bits and Channel Points are visible without the Bonus Probably should be visible with the Bonus active as well

**Screenshots**
![Twitch](https://github.com/twitchdev/issues/assets/78897688/7b103e25-40a7-4b43-9a9c-0efdfdc0202e)
![Twitch2](https://github.com/twitchdev/issues/assets/78897688/82834550-e3a8-4d8f-9ef4-a17a50a96f68)

**Additional context or questions**
My understanding is that it is probably worked on under latest update and could have been missed.

**Comment by BarryCarlyon** [comment]
This tracker is for third party developer issues.

Issues with first party should be flagged with Twitch Support.

TLDR: this has been reported in the wrong place
